bert_e/tests/test_bert_e.py

```diff
@@ -569,7 +569,7 @@ def test_branch_cascade_mixed_versions(self):
             2: {'name': 'development/4.3', 'ignore': True},
             3: {'name': 'development/5.1.8', 'ignore': True},
             4: {'name': 'development/5.1', 'ignore': False},
-            5: {'name': 'development/10', 'ignore': False}
+            5: {'name': 'development/10.0', 'ignore': False}
```

Comment:
Do we test somewhere the dev/x (one digit) only? I don't want to lose this test ;)